src/config/env.js

```diff
@@ -0,0 +1 @@
+require('dotenv').config()
```

Comment:
instead of creating a new file we can directly use this line wherever needed

Reply:
yup we can , just followed the convention